File: code/simian/simian-master-v1.6/SimianJS/Tests/test.hash.lua
#!/usr/bin/env luajit
local hash = require ("hash")

local COUNT = 10000000

local start = os.clock()
local h = 0
for i=0,COUNT-1 do h = hash(tostring(i)) end
local finish = os.clock() - start
print(h);
print("Took: " .. finish*1000 .. " ms");

File: code/simian/simian-master-v1.6/SimianJS/Tests/test.hash.js
//#!/usr/bin/env js --enable-avx
const load = masala.io.load, print = masala.io.print, now = masala.time.now;

load("./hash.js");

const COUNT = 10000000;

var start = now();
var h = 0;
for (var i=0; i<COUNT; i++) h = hash(i.toString());
var finish = now() - start;
print(h);
print("Took: " + finish + " ms");
